web/styles/typing_notifications.css

```diff
@@ -9,3 +9,9 @@
     list-style: none;
     margin: 0;
 }
+
+#mark_as_read_turned_off_banner {
+    /*  override the margin of main-view-banner since it causes gap
```

Comment:
`causes gap` -> `causes more gap than we want`

Also, looks like we need this for other banners as well which are displayed in the message feed.

Reply:
> causes gap -> causes more gap than we want

updated
> Also, looks like we need this for other banners as well which are displayed in the message feed.
there are only some indicators (no banners)

 other than the `mark_read_on_scroll_state_banner` there's only `scheduled_message_indicator` below `message-lists-container`, which deosn't seem to impact it anyways.

Reply:
I see, thanks!